cursorless-talon/src/modifiers/delimiter_range.py

```diff
@@ -0,0 +1,23 @@
+from typing import Any
+
+from talon import Context, Module
+
+mod = Module()
+ctx = Context()
+
+
+delimiter_ranges = {
+    "leading": {"direction": "leading"},
+    "trailing": {"direction": "trailing"},
+}
+
+mod.list("cursorless_delimiter_range", desc="Types of delimiter ranges")
+ctx.lists["self.cursorless_delimiter_range"] = delimiter_ranges.keys()
+
+
+@mod.capture(rule="{user.cursorless_delimiter_range}")
+def cursorless_delimiter_range(m) -> dict[str, Any]:
+    return {
+        "type": "delimiterRange",
```

Comment:
Not in love with this identifier. "Range" is a bit confusing. Tbh I'd be tempted to just have two separate types `leadingDelimiter` and `trailingDelimiter`. Unless we can come up with a better name here

Reply:
Since they are handled by the same modifier stage I see no point in splitting them into two different types. I think delimiter range is clear enough. Is either the leading delimiter range or the trailing delimiter range that is the nomenclature we are using all over the place in the extension isn't it?